### ISSUE TITLE
'win' 'winChances' and 'dangerLvl' all unresolved references

### ISSUE THREAD

**Issue by DovahBeat** [dnglokpor/IsekaiBot#3]
<img width="335" alt="Screen Shot 2024-08-20 at 10 54 17 AM" src="https://github.com/user-attachments/assets/b896a1cd-7ca0-4e3f-9629-83989361868e">
Hello there, 
I know you haven't updated or been interested in updating for quite some time, but I wanted to check this bot code out for my own rpg server I'm making.

Anyways, I downloaded all the files and put them into my project file, and followed the read me instructions, but I get errors in the IsekaiBot.py file. They say "Unresolved reference 'win'", Unresolved reference 'winChances'", and Unresolved reference 'dangerLvl'", I don't know why or what i possibly did wrong.

Thank you for all your work on this bot, making it open source, and offering support!

**Comment by dnglokpor** [comment]
Hi @DovahBeat I see you marked this a completed. Did you figure out what the issue was?
Sorry this project was started half out of boredom during the pandemic so and I posted it on here to showcase python as a skill while I was in college. I never really maintained it. My objective was to get some understanding of the Discord APIs. Frankly the system on this is too hard to use and a bit clunky. I wouldn't really know where to start to help you.

I made this an [alt version](https://github.com/dnglokpor/IdleFantasyHub) that I think makes better use of a linux server environment and has some amount of visuals added as well. Maybe this would work better for you? I can offer better support on this one since it's the most recent iteration on the concept.
